app/src/main/java/io/gnosis/safe/ui/transaction/TransactionView.kt

```diff
@@ -58,7 +58,7 @@ sealed class TransactionView(open val status: TransactionStatus) {
         val nonce: String
     ) : TransactionView(status)
 
-    data class ChangeMastercopy(
+    data class SettingsChangeVariant(
```

Comment:
why such strange name?

Reply:
Because it is used for more than only change mastercopy. Do you have a better name. It is used for enable/disable module, change mastercopy and change fallback handler.